src/test/ui/consts/const-eval/issue-50814.rs

```diff
@@ -14,12 +14,15 @@ struct Sum<A,B>(A,B);
 impl<A: Unsigned, B: Unsigned> Unsigned for Sum<A,B> {
     const MAX: u8 = A::MAX + B::MAX;
     //~^ ERROR any use of this value will cause an error [const_err]
+    //~| ERROR any use of this value will cause an error
     //~| WARN this was previously accepted by the compiler but is being phased out
+    //~| WARNING this was previously accepted by the compiler
```

Comment:
```suggestion
    //~| WARN this was previously accepted by the compiler but is being phased out
```
keep the two lines consistent